run_scream.syrah_ne30_standard.csh

```diff
@@ -0,0 +1,1456 @@
+#! /bin/csh -fe
```

Comment:
I don't think you meant to include this file in the PR.

Reply:
Also for the other run_scream file you added too.

Reply:
Deleted. thought I deleted it last time.